use_case_guide/multi_tenant.rst

```diff
@@ -230,7 +230,7 @@ The next step is loading sample data into the cluster from the command line.
 
 Being an extension of PostgreSQL, Citus supports bulk loading with the COPY command. Use it to ingest the data you downloaded, and make sure that you specify the correct file path if you downloaded the file to some other location. Back inside psql run this:
 
-.. code-block:: psql
+.. code-block:: postgres
```

Comment:
This change is unfortunate because it removes highlighting from all lines. However I could not figure out how to make the psql lexer (aka [PostgresConsoleLexer](https://www.complang.tuwien.ac.at/doc/python-pygments/lexers.html#lexers-for-various-sql-dialects-and-related-interactive-sessions)) color all the `\copy` commands. The only way I could discover was to break the block into multiple little blocks, each with a single copy in them, but that looked bad.